include/uk/ctors.h

```diff
@@ -42,7 +42,7 @@
 extern "C" {
 #endif
 
-typedef void (*uk_ctor_func_t)(void);
+typedef void (*uk_ctor_func_t)();
```

Comment:
Why no arguments? Shouldn't it be the types of `argc`/`argv`?

Reply:
To make it compatible with any number of function arguments even though nothing is handed over.